starters/cms/src/main.tsx

```diff
@@ -31,3 +24,18 @@ if (didSync) {
         </StrictMode>
     )
 }
+
+if (framer.mode === "syncManagedCollection") {
+    if (framer.isAllowedTo(...syncMethods)) {
+        const { didSync } = await syncExistingCollection(activeCollection, previousDataSourceId, previousSlugFieldId)
+        if (didSync) {
+            await framer.closePlugin("Synchronization successful", { variant: "success" })
+        } else {
+            launch()
```

Comment:
This conditional tree is pretty hard to read. How about we check for permissions within `syncExistingCollection`?

Reply:
Moved to `syncExistingCollection`!